apps/bond/components/BondingProducts/Bonding/BondingList/BondingList.jsx

```diff
@@ -240,10 +240,22 @@ const sortProducts = (list) =>
     // - the API returns zero (shouldn't happen) OR
     // - has error OR
     // - not fetched yet
-    if (isCurrentPriceLpZero(a.fullCurrentPriceLp)) return 1;
```

Comment:
previous sorting placed profitable products after not loaded if they were in the middle